Fixed_Income_and_Debt/convexity/convexity.Rmd

```diff
@@ -9,11 +9,11 @@ runtime: shiny
 
 ## Bond Convexity
 
-Bond convexity refers to the actual convex (non-linear) relationship between a bond's price and yield.  This is stated in relation to the linear [duration](https://micfm.shinyapps.io/intro_duration/) approximation of the bond price and yield relationship.
+Bond convexity refers to the actual convex (non-linear) relationship between a bond's price and yield.  This is stated in relation to the linear [duration](http://www.5minutefinance.org/concepts/duration) approximation of the bond price and yield relationship.
 
--  Convexity is often used as a general term for the error when using duration to approximate interest rate risk.
+-  Convexity is often used as a general term for the error made when using duration to approximate interest rate risk.
```

Comment:
Ah, makes sense! Let's say, "...a general term for the _appoximation_ error when using duration to approximate interest rate risk..." 

Sound okay?